src/Makefile.am

```diff
@@ -1,6 +1,7 @@
 CONFIG_CLEAN_FILES = *.loT *.rej *.orig
 AM_CPPFLAGS = -I${top_srcdir} -I${top_srcdir}/src
 sbin_PROGRAMS = fapolicyd
+bin_PROGRAMS = fapolicy
```

Comment:
Why are we changing this to be a bin program instead of sbin? Traditionally, daemons live in sbin because normal users cannot run the program. Meaning they need privileges to work correctly.

Reply:
fapolicy is CLI binary. That should be in bin or not?

Reply:
I was going to mention that I do not see where the fifo is created by fapolicyd. Can't find it anywhere. I am wanting to see what the permissions are for the fifo. Would it be 0600? If so, then you need to be admin and it makes sense to put the cli in sbin. But also, the name is so close to the daemon that one could mistake it for the daemon. Might suggest fapolicyd-cli or fapolicyd-notify or something else.

Reply:
https://github.com/linux-application-whitelisting/fapolicyd/blob/1e45cf440e5bcb63e29322b72811f81b0ed271d7/src/database.c#L739

Reply:
I think it should be 0600.